xla/service/gpu/kernels/cutlass_gemm_custom_kernel.cc

```diff
@@ -101,7 +101,11 @@ KernelArgsPacking ArgsPacking(int32_t m, int32_t n, int32_t k,
   // object constructed in the storage. For now we ignore it, and it's textbook
   // definition of UB, but for CUTLASS kernels we use today it's perfectly safe.
   struct Params {
+#if defined(_MSC_VER)
+    alignas(64) std::byte storage[1024];
+#else
     alignas(128) std::byte storage[1024];
+#endif
```

Comment:
This results in the following error on Windows:
```
error C2719: '<args_0>': formal parameter with requested alignment of 128 won't be aligned
```
cc @dimvar who previously made the change from `64` to `128`.